### ISSUE TITLE
Undefined offset Error when result exists

### ISSUE THREAD

**Issue by dbr0** [teamtnt/laravel-scout-tntsearch-driver#30]
Hi, I'm getting this error on my deployed server but not on my local machine (deploying by Forge@DigitalOcean, locally I'm using Homestead).

The error log:

>  in Collection.php line 1274
> at HandleExceptions->handleError('8', 'Undefined offset: 209', '/home/forge/mysite.hr/vendor/laravel/framework/src/Illuminate/Support/Collection.php', '1274', array('key' => '209')) in Collection.php line 1274
> at Collection->offsetGet('209') in TNTSearchEngine.php line 158
> at TNTSearchEngine->TeamTNT\Scout\Engines\{closure}('209', '0')
> at array_map(object(Closure), array('209'), array('0')) in Collection.php line 597
> at Collection->map(object(Closure)) in TNTSearchEngine.php line 159
> at TNTSearchEngine->map(array('ids' => array('209'), 'execution time' => '0.6032 ms'), object(Element)) in Engine.php line 62
> at Engine->get(object(Builder)) in Builder.php line 115
> at Builder->get() in ControllerInWhichICallTheSerachMethod.php line 57

Depending on the size of the string the error will be `Undefined offset: 209` or `Undefined offset: 205` and so on.

Please do ask if you have any questions.

**Comment by nticaric** [comment]
Is it possible that you deployed the index from your local machine also? So there might be some ids that don't exist on the deployed server. What you can do is to run

`php artisan tntsearch:import App\\Model` on your server to recreate the index

**Comment by dbr0** [comment]
That was it. Thanks!

For any other noob like me that comes up to this:

I ran `ph

- p artisan tntsearch:import App\\Model` and then ran `git status` on my local machine to find where the index file was created. It is in the `/storage` folder.
- Then I updated my `.gitignore` file and added new line `/storage` since I concluded there was no point in uploading that folder to the server.
- Next I ran `git add -A && git commit -m 'removed /storage folder from my commits` && git push
- Next I SSHed into my ser

ver using `ssh forge@mysite.hr` and ran `php artisan tntsearch:import App\\Model`

And now it works.
